views/search.php

```diff
@@ -140,14 +140,27 @@
 
 	<div id="string-locator-search-results-table-wrapper" class="<?php echo esc_attr( implode( ' ', $wrapper_classes ) ); ?>">
 		<div class="tablenav top">
-			<?php
-			/**
-			 * An action to output controls in the tablenav region, which only
-			 * become visible when there are search results available.
-			 */
-			do_action( 'string_locator_search_results_tablenav_controls' );
-			?>
-			<br class="clear" />
+			<div style="display: flex;">		
```

Comment:
The flex style here makes the replacement controls be limited in width and looks "wrong" when expanding the replacement panel.